internal/install/recipe_file.go

```diff
@@ -6,11 +6,11 @@ type recipeFile struct {
 	Install        map[string]interface{} `yaml:"install"`
 	InstallTargets recipeInstallTarget    `yaml:"installTargets"`
 	Keywords       []string               `yaml:"keywords"`
-	MELTMatch      meltMatch              `yaml:"meltMatch"`
```

Comment:
Last conversation we had, I thought this was remaining.  Am I out of date here?

Reply:
You are right, @jbeveland27 says it's staying for now.  For some reason it was causing panics when marshaling to YAML the way it was set up before.  I'll put it back in place.